openedx/core/lib/cache_utils.py

```diff
@@ -159,6 +159,7 @@ class CacheInvalidationManager:
 
     To use, instantiate with a namespace or django model class:
     `manager = CacheInvalidationManager(model=User)`
+    One of namespace or model should be specified, but not both.
```

Comment:
I think the current behavior is not great, but I'm constraining myself to just leaving a warning in the docstring.